Library/Formula/freeswitch.rb

```diff
@@ -1,7 +1,8 @@
 class Freeswitch < Formula
-  homepage "http://freeswitch.org"
-  url "https://stash.freeswitch.org/scm/fs/freeswitch.git", :tag => "v1.4.15"
-  head "https://stash.freeswitch.org/scm/fs/freeswitch.git"
+  homepage "https://freeswitch.org"
+  url "https://freeswitch.org/stash/scm/fs/freeswitch.git", :tag => "v1.4.19", :revision => "ebf2df68fa6f341311620c6d90f5a8d77334606c"
```

Comment:
Can we shunt this onto a few lines so the formulae doesn't lose easy readability?

```ruby
url "https://freeswitch.org/stash/scm/fs/freeswitch.git",
    :tag => "v1.4.19",
    :revision => "ebf2df68fa6f341311620c6d90f5a8d77334606c"
```

Should be fine.